### ISSUE TITLE
Campaign Settings->Campaign Cover Image can load images, but not save them

### ISSUE THREAD

**Issue by oldmicah** [worldanvil/worldanvil-bug-tracker#1049]
Oldmicah

**no world / Kingdom of Altonne /**:

Appears to be a bug

Not public

**Describe the Issue**
For Campaign Settings->Campaign Cover Image I can load images, but when I hit save they disappear. (and don't load)

**To Reproduce**
1. Go to Dashboard->Campaign->Settings->Campaign Cover Image
2. Drag a file (or browse, same behavior) to the preview square3. 
3. Wait for image upload and appear.  Has the 'remove' label on it.4. 
4. Click on the 'Save Changes' button5. 
5. Image dissapears from the preview rectangle6. 6. 
       Does appear in Dashboard->Images though
       I can upload this same file directly to Dashboard->Images
      I have been able to upload and save images in other parts of the program
      This happens with .jpg files and .png files.  I've also tried completely different images with same result.

**Expected behavior**
Like in other parts of the program, I expect the image to stay in the preview rect and be visible when I load relevant pages.   Appreciate the help and apologies if I'm doing something wrong and didn't realize it.  Let me know if I can test anything else or provide more info.

**Screenshots**
The two screenshots below are from after I load the image and after I hit the save. (empty preview rect)

<img width="1544" alt="Screen Shot 2022-05-23 at 7 58 32 PM" src="https://user-images.githubusercontent.com/242680/169927959-951977dd-96c9-4d24-a980-8a55c2592bdb.png">

<img width="1572" alt="Screen Shot 2022-05-23 at 7 58 56 PM" src="https://user-images.githubusercontent.com/242680/169927964-f3965c53-8c92-4173-97a8-1870425b4ba4.png">

**Comment by Kordishal** [comment]
I believe there might be a 1 MB limit on that image.

Alternatively JPG / PNG can have metadata that can't be handled by WA. Try converting to WEBP!

**Comment by oldmicah** [comment]
Hmmm... Either time or your touching the page fixed it.  It was working this morning.  

For he record, the image was small ~156k (verified.). I couldn't quickly figure out how to upload a WEBP, but I did try generating the jpg through another program.  (screen-shotted it in, so no meta data).  Same result.  

My guess is that it's just taking a long time to process. (at least 6 hours as I started at 4pm and checked it at 10)  Pleased that it worked, but delays like this make users flounder and get frustrated b/c they think they just aren't getting the fairly complex UI.  Having better descriptors on the fields (or better yet going to a sample page with the parts labeled) would help a lot.

I seriously do appreciate your help.  Thank you!
